swagger.js

```diff
@@ -32,12 +32,64 @@ const doc = {
         },
       },
     },
-    TokenizedUser: {
-      user: {
-        name: 'John Doe',
-        userId: '64ef5885c2a489217c571253',
-        role: { '@enum': ['user', 'admin'] },
+
+    Order: {
+      $_id: 'idabc123456',
+      $user: {
+        $id: 'idabc123456',
+        $name: 'John Doe',
+        $username: 'johndoe',
+        $email: 'john@example.com',
+        $role: { '@enum': ['user', 'admin'] },
+      },
+
+      $orderStatus: {
+        '@enum': [
+          'pending',
+          'payment_successful',
+          'payment_failed',
+          'cancelled',
+          'complete',
+        ],
+        default: 'pending',
       },
+      orderItems: [
+        {
+          $album: '5f7f1f7a5c3f2b2d9c1b0b9d',
+          $quantity: 2,
+          $_id: 'idabc12345',
+        },
+      ],
+      $subtotal: 9.99,
+      $tax: 0.75,
+      $total: 10.74,
+      $createdAt: '2023-08-08T20:45:00.942Z',
+      $updatedAt: '2023-08-08T20:45:00.942Z',
+      paymentIntentId: 'pi_abc123456',
+    },
+
+    NewOrder: {
+      $subtotal: 100,
+      $tax: 0.15,
+      $total: 115,
+      $orderItems: [{ album: 'abcdef12345', quantity: 2 }],
+    },
+
+    OrderList: {
+      orders: [
```

Comment:
In addition to the `orders` list, the `getAllOrders` endpoint also returns a field `count` [https://github.com/Code-the-Dream-School/dd-prac-team4-back/blob/main/src/controllers/orderController.js#L64](https://github.com/Code-the-Dream-School/dd-prac-team4-back/blob/main/src/controllers/orderController.js#L64)

![](https://github.com/Code-the-Dream-School/dd-prac-team4-back/assets/9094098/a8e5b8ae-8473-4351-a609-6573857fc762)

We should include that in our definition here:

```suggestion
      count: 1,
      orders: [
```